src/components/counter/Counter.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import "./Counter.scss";
+
+class Counter extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { value: 0 };
+  }
+
+  changeValue(increase) {
+    const newValue = increase ? this.state.value + 1 : this.state.value - 1;
```

Comment:
@Marysosh Could it be simplify like `this.state.value++` and `this.state.value--`

Reply:
Sure. In order to avoid direct state mutation, I've added a check for this condition inside the function setState. Corrected.